imports/plugins/core/versions/server/migrations/15_update_shipping_status_to_workflow.js

```diff
@@ -0,0 +1,78 @@
+import { Migrations } from "meteor/percolate:migrations";
+import { Orders } from "/lib/collections";
+
+Migrations.add({
+  version: 15,
+  // Reaction v1.0 had 3 shipping boolean states (packed, shipped, delivered). Shipping workflow is not managed with a
+  // workflow object that keeps track of previous state.
+  up: function () {
+    Orders.find().forEach((order) => {
+      const currentShipping = order.shipping[0];
```

Comment:
Should this be the hard-coded first shipping object or the shipping object associated with the current active shop?

Reply:
I did that to staying consistent with 1.0 (or below 1.5) codebase. It's all x[0] in there. Since this migration file is interacting with data created from such code, I referenced it that way. 
I think it's more recently that we started referencing shipping it by shopId (i.e marketplace).